src/client/components/homeComponents/HomeRatings.jsx

```diff
@@ -0,0 +1,16 @@
+import pikachuImg from "../../assets/pikachuTest.png";
+
+const HomeRatings = () => {
+    return (
+        <div className="ratingContainer">
```

Comment:
since this is in its own component, fetching and mapping data will be much easier to implement